tests/functional/raw-node/test/routes/routeBackbeat.js

```diff
@@ -172,6 +189,155 @@ describeSkipIfAWS('backbeat routes', () => {
     });
 
     describe('backbeat PUT routes', () => {
+        describe('null version', () => {
+            beforeEach(done => s3.createBucket({ Bucket: BUCKET_FOR_NULL_VERSION }, done));
+            afterEach(done => {
+                bucketUtil.empty(BUCKET_FOR_NULL_VERSION)
+                    .then(() => s3.deleteBucket({ Bucket: BUCKET_FOR_NULL_VERSION }).promise())
+                    .then(() => done());
+            });
+
+            it('should update metadata of a current null version', done => {
```

Comment:
Would you mind adding some similar tests on versioning-suspended buckets:

- when there is an existing null version
- when there is no existing null version

and make sure we only delete the data when a prior null version exists

Reply:
In S3C, the use of the "putMetadata" call is restricted to versioned buckets.
I will add tests for Zenko/Artesca branches that support PUT metadata for non-versioned buckets.